src/python/pants/backend/python/goals/lockfile.py

```diff
@@ -343,7 +343,7 @@ async def python_lockfile_synthetic_targets(
                     TargetAdaptor(
                         "_lockfiles",
                         name=synthetic_lockfile_target_name(name),
-                        sources=[lockfile],
+                        sources=(lockfile,),
```

Comment:
Not strictly necessary, since we will deep_freeze() the sources kwarg, but it's good to model the right behavior...